src/lib/validators.js

```diff
@@ -12,4 +12,12 @@ export function positive(value) {
 
 export function address(value) {
     return /^0x[a-fA-F0-9]{40}$/.test(value) ? undefined : "Not an address"
+}
+
+export function password(value) {
+    return ((value) && (value.length >= 8)) ? undefined : "Please enter at least 8 characters"
```

Comment:
You can return another function from here, so you can create something like:
```
export function minLength(min) {
   return (value) -> {
     return value.length >= min ? undefined : "Please enter at least " + min + " characters"
   }
}
```
and use like `validate={[required, minLength(8)]}`